UPGRADE-6.7.md

```diff
@@ -1,20 +1,86 @@
+When you use default stores in your plugin you need to switch from `Shopware.State` (Vuex) to `Shopware.Store` (Pinia).
+Adding your own Vuex stores is still possible, however it is recommended that you switch to Pinia as well.
+
+**TBD, can we have a realistic example how this looked like before and what actually needs to change?
+
+For more information refer to the [docs](https://developer.shopware.com/docs/resources/references/adr/2024-06-17-replace-vuex-with-pinia.html#replace-vuex-with-pinia).
+
+# Cache Rework
+**Note:** Those changes can be activated separately with the `cache_rework` feature flag.
+
+## Delayed Cache Invalidation
+The cache invalidation will be delayed by default. This means that the cache will be invalidated in regular intervals and not immediately.
+This will lead to better cache hit rates and way less (duplicated) cache invalidations, which will improve efficiency and scalability of the system.
+As this feature is now active by default the previous `shopware.cache.invalidation.delay` configuration is removed.
+
+The default interval is 5 min, this can be changed by adjusting the run interval of the `shopware.invalidate_cache` scheduled task.
+
+If you sent an API request with critical information, where the cache should be invalidated immediately, you can set the `sw-force-cache-invalidate` header on your request.
+```
+POST /api/product
+sw-force-cache-invalidate: 1
+```
+
+To manually clear all the stale caches you can either run the `cache:clear:delayed` command or use the `/api/_action/cache-delayed` API endpoint.
+```
+bin/console cache:clear:delayed
+```
+```
+DELETE /api/_action/cache-delayed
+```
+
+For debugging there is the `cache:watch:delayed` command available, to watch the cache tags that are stored in the delayed cache invalidation queue.
+```
+bin/console cache:watch:delayed
+```
+
+## Removal of Store-API route caching
+The Store-API route caching has been removed. This means that the `Cached*Route` classes will be removed.
+This solves some weird states when the HTTP-Cache was invalidated separately from the route cache.
+Additionally, the cache hit rate for the Store-API was low, so the performance impact should be minimal, but the amount of cache items and cache invalidations will be reduced.
+This overall should lead to more effective cache resource usage.
+
+## Use ESI includes for Header and Footer
+**TBD**
+
+# Major Library Updates
+We upgraded the following libraries to their latest versions:
+* [DBAL 4.x](https://github.com/doctrine/dbal/blob/4.2.x/UPGRADE.md#upgrade-to-40): When you are using DBAL directly, please check the upgrade guide.
+* [PHPUnit 11.x](https://github.com/sebastianbergmann/phpunit/blob/11.0.0/ChangeLog-11.0.md#1100---2024-02-02): You need to adjust your tests to the new PHPUnit version.
+* [Dompdf 3.x](https://github.com/dompdf/dompdf/releases/tag/v3.0.0): Please check your document templates, if they are still rendered as expected.
+
+# Accessibility Compliance
```

Comment:
@shopware/ct-storefront can you double check a11y related changes here?

Reply:
Looks like the focus states are missing: 

https://github.com/shopware/shopware/blob/trunk/changelog/_unreleased/2024-12-16-unify-focus-outlines-for-better-accessibility.md

https://github.com/shopware/shopware/pull/5881

But I really like this structure. 👍 It is a way better to read and filter after the stuff that effects me.

Reply:
ah it's missing because it is not yet released, we need to put the stuff that is currently in the changelogs into the correct places after our automation ran after tagging the release, then we will incorporate that -> don't want to do it now for fear that we will end up having that info twice